net/node_exporter/Portfile

```diff
@@ -3,6 +3,7 @@ PortGroup           github 1.0
 
 github.setup        prometheus node_exporter 0.18.1 v
 github.tarball_from archive
+revision            1
```

Comment:
As nothing is changing in the installed files, a revision bump is not required.

Reply:
OK that makes sense, fixed.